src/raftstore/store/metrics.rs

```diff
@@ -213,10 +213,19 @@ lazy_static! {
             "Bucketed histogram of rocksdb ingestion durations",
             exponential_buckets(0.005, 2.0, 20).unwrap()
         ).unwrap();
+
     pub static ref RAFT_INVALID_PROPOSAL_COUNTER_VEC: IntCounterVec =
         register_int_counter_vec!(
             "tikv_raftstore_raft_invalid_proposal_total",
             "Total number of raft invalid proposal.",
             &["type"]
         ).unwrap();
+
+    pub static ref RAFT_EVENT_TIMEOUT: HistogramVec =
+        register_histogram_vec!(
+            "tikv_raftstore_event_timeout",
+            "Raft store each event timeout",
```

Comment:
It is duration instead of timeout. 
`"Duration of raft store events."`.

Reply:
Agree.